src/tests/ftest/launch.py

```diff
@@ -1281,6 +1320,10 @@ def main():
         "-s", "--sparse",
         action="store_true",
         help="limit output to pass/fail")
+    parser.add_argument(
+        "-sec", "--secure_mode",
+        action="store_true",
```

Comment:
My recollection of _argparse_ is fuzzy, but doesn't `store_true` mean the default value for the option is `False`?  We should be **secure by default**.  That's an SDL principle.

Reply:
Please see my comment above.
In this phase, -sec as an option.

Reply:
It shouldn't be.  _Insecure_ should be the option.  _Secure_ should be the default.

Reply:
The secure-mode by default will be after the pilot tests stabilized in the CI.